<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicImageSelect</class>
 <widget class="QDialog" name="BasicImageSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>281</width>
     <height>673</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="labelImageName">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(140, 140, 140);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>图像列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>626</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetImage">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>41</y>
      <width>261</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>626</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBig">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>17</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>大图</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSmall">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>17</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>小图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>626</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
